com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -121,6 +122,7 @@ private void DidStart(string error)
         ChartboostMediation.SetSubjectToGDPR(false);
         ChartboostMediation.SetSubjectToCoppa(false);
         ChartboostMediation.SetUserHasGivenConsent(true);
+        ChartboostMediation.SetTestMode(true);
```

Comment:
May want to see if there's a canary setting to get this from the app. In case, QE wants to test without test mode on.

Reply:
Canary will have a button for it. This is only for the demo scene that. I was going to take that out later but I can just do it now